features/step_definitions/admin_steps.rb

```diff
@@ -253,7 +253,7 @@
   uncheck("user_roles_#{role_id}")
 end
 
-When (/^I make a translation of an admin post( with tags)?$/) do |with_tags|
+When (/^I make a translation of an admin post( with tags "([^\"]*)")?$/) do |tags|
```

Comment:
Yep, I remember we have the Cucumber Expressions guideline, but I believe it would not work here.

I investigated the same problem a couple months back and I didn't find a way to use Cucumber Expressions for optional parameters. Optional text is supported, required parameters are supported, but it doesn't seem like both combined are.
If someone can prove me wrong, I would be very happy.

Line 147 has no optional parameters, so it should be possible to use `{sting}` there. I am not sure if mix of styles will be good in seemingly related functions, but can change.

`"([^\"]*)"` regex itself probably can be updated to `"(.*?)"`. I originally decided to just go with the nearby code in the file because I have no idea why backslash was added here in the first place.

Reply:
In that case, just making the change from `"([^\"]*)"` to `"(.*?)"` sounds good!

Reply:
I think because Cucumber Expressions are deliberately simpler, we would need to use multiple step definitions instead of one with optional clauses.

Staying with regex in this case sounds good to me, however.